src/nav/components.yml

```diff
@@ -102,6 +98,8 @@
       id: /components/tiles
     - title: Tooltip
       id: /components/tooltip
+    - title: Tooltip Modal
```

Comment:
This one can be `Tooltip modal` for UXCS standards

Reply:
Updated.